clang/docs/ReleaseNotes.rst

```diff
@@ -245,6 +245,12 @@ Modified Compiler Flags
        f3 *c = (f3 *)x;
      }
 
+- Added a new diagnostic flag ``-Wpointer-integer-ordered-compare`` which is
+  grouped under ``-Wpointer-integer-compare`` and moved previously
+  ungrouped diagnostics ``ext_typecheck_ordered_comparison_of_pointer_integer``,
+  ``ext_typecheck_ordered_comparison_of_pointer_and_zero`` under
+  ``-Wpointer-integer-ordered-compare``. Fixes #GH88090
```

Comment:
Maybe add a sentence to say that this also resolves a false negative for comparison between pointers and literal zero.

Reply:
done